### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by umair-rao**
In this PR

 

- [ ] Create a database named vet_clinic.

 

- [ ] Create a table named animals with the following columns:

id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
create table

- [ ] Insert the data into the table
insert data

- [ ] Query to find all animals whose name ends in "mon".
mon

- [ ]  Query to List the name of all animals born between 2016 and 2019.
date

- [ ]  Query to list the name of all animals that are neutered and have less than 3 escape attempts.
3 attem

- [x] Query to list the date of birth of all animals named either "Agumon" or "Pikachu".
Agumon dat of borth

- [ ] Query to list name and escape attempts of animals that weigh more than 10.5kg
weight

- [ ] Query to find all animals that are neutered.
neutered

- [ ] Query to find all animals not named Gabumon.
Gabunon

- [ ] Query to find all animals with a weight between 10.4kg and 17.3kg

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
-CREATE TABLE animals (
-    name varchar(100)
-);
+vet_clinic=# CREATE TABLE animals (
+vet_clinic(# id BIGSERIAL NOT NULL PRIMARY KEY,
+vet_clinic(# name CHAR(20) NOT NULL,
+vet_clinic(# date_of_birth DATE NOT NULL,
+vet_clinic(# escape_attempts INT,
+vet_clinic(# neutered BOOLEAN,
+vet_clinic(# weight_kg DECIMAL );
```
Comment (Munsa1) [inline schema.sql]: - [ ] (Optional). You could add your code usig the format below for readability purposes
image

File: README.md [hunk 2]
```diff
@@ -1,286 +1,82 @@
...
-<!-- TABLE OF CONTENTS -->
+# 📖 [VET CLINIC]
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
```
Comment (Munsa1) [inline README.md:48]: - [ ] Kindly readd the `Key Features` section, `Future Features Section` with the appropriate information for each section

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @umair-rao, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (umair-rao) [reply schema.sql]: File updated.

File: README.md (hunk 2 above)
Comment (umair-rao) [reply README.md:48]: As Future Features are not mentioned anywhere therefore I skipped this.

**Review by Deepakdanger — APPROVED** [review]
Hi @umair-rao , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly add a key feature section with the project feature in the readme file as requested by the previous reviewer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**